packages/server-wallet/scripts/inject-wallet-version.js

```diff
@@ -0,0 +1,10 @@
+var fs = require('fs');
+
+fs.writeFileSync(
+  'lib/src/version.js',
+  `"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.WALLET_VERSION = void 0;
+exports.WALLET_VERSION = '${process.env.npm_package_name}@${process.env.npm_package_version}';
```

Comment:
Reads env vars that exist during lifecycle scripts.

Reply:
What's the advantage of putting the `npm_package_name` here? Won't that always be the same?

Reply:
It's for the benefit of any other wallet that this one may wish to communicate with: for example our very own `xstate-wallet` or some third party nitro wallet.

Reply:
We could hard-code the package name into this file, but this has the benefit of being robust to changing the package name.

Reply:
If I am publishing a canary build would this pick up that version e.g., `next.ab423+72.ab423`

Reply:
I believe that it would